base/applications/mscutils/eventvwr/lang/es-ES.rc

```diff
@@ -197,7 +197,7 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_COPY "Event Type:     %s\r\n\
+    IDS_COPY "Tipo de evento:     %s\r\n\
```

Comment:
You should remove exactly 4 spaces after the semicolon and before the '%s', so as to keep all the formatted string correctly aligned:
```
Tipo de evento: %s\r\n\
Event Source:   %s\r\n\
Event Category: %s\r\n\
```
(by the way the other substrings in it are not yet translated?)

Reply:
Fixed. I had little mistakes because I was a bit inattentive.